src/cwt.rs

```diff
@@ -0,0 +1,114 @@
+use serde::{Deserialize, Serialize};
+use serde_cbor::Value;
+use std::collections::BTreeMap;
+
+pub use crate::claim::{self, Claim, Label, NumericDate};
+
+/// CWT claims set.
+#[derive(Clone, Debug, Default, Deserialize, Serialize, PartialEq, Eq)]
+pub struct ClaimsSet(BTreeMap<Value, Value>);
+
+impl ClaimsSet {
+    pub fn insert_claim<T: Claim>(&mut self, claim: T) -> Option<Value> {
+        self.0.insert(T::label().into(), claim.into())
+    }
+
+    pub fn get_claim<T: Claim>(&self) -> Option<&Value> {
+        self.0.get(&T::label().into())
+    }
+
+    pub fn serialize(&self) -> Result<Vec<u8>, Error> {
```

Comment:
```suggestion
    /// Serialize the ClaimsSet to CBOR bytes, so that it 
    /// can be attached as a payload to a COSE object.
    pub fn serialize(&self) -> Result<Vec<u8>, Error> {
```